frontend/packages/portal-frontend/src/resources/components/ResourcesPage.tsx

```diff
@@ -67,7 +101,9 @@ export default function ResourcesPage(props: ResourcesPageProps) {
                       <ListGroupItem
```

Comment:
Another thing to check. Are these links triggering the Accordion's internal onClick event?